test/testlib/OpenSSL/Test.pm

```diff
@@ -937,45 +1025,25 @@ sub __fixup_cmd {
     return undef;
 }
 
-sub __build_cmd {
+# __decorate_cmd NUM, CMDARRAYREF
+#
+# __decorate_cmd takes a command number NUM and a command token array
+# CMDARRAYREF, builds up a command string from them and decorates it
```

Comment:
I added a documentation comment in the section about `cmd` and its derivatives, since that's where that complexity is implemented.